st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -112,6 +114,51 @@
     pass
 
 
+def get_config_details(yaml_file_name, section_name):
+    """
+    To get the configurations from st2 config file.
+    :param yaml_file_name: config yaml file name
+    :section_name: option to get config from yaml file
+    :return: requested option from config file
+    :rtype: ``str`` or ``list``
+    """
+    with open(yaml_file_name, 'r') as yaml_file:
+        conf = yaml.load(yaml_file)
+    if section_name == 'log_file_path':
```

Comment:
There seems to be a lot of similar code in this function - it looks like this could be solved by defining a module level dictionary, something like:

```python
DEBUG_INFO_CONFIG_FILE_OPTIONS = {
   'section_name': {
       'option_name': {
          'default_value': None,
          'tranform_func': lamda value: value  # this would only be used for values which need additional processing (e.g. st2_config_file_path)
       }
   }
}
```

And then using for loop in this function.

I believe we already do similar thing in some other place (perhaps in st2client).